doc/UsersGuide/BuildingRunningTesting/ContainerQuickstart.rst

```diff
@@ -201,77 +200,58 @@ Generate the Forecast Experiment
 =================================
 To generate the forecast experiment, users must:
 
-#. :ref:`Activate the workflow <SetUpPythonEnvC>`
+#. :ref:`Stage the container <SetUpCont>`
 #. :ref:`Set experiment parameters to configure the workflow <SetUpConfigFileC>`
 #. :ref:`Run a script to generate the experiment workflow <GenerateWorkflowC>`
 
 The first two steps depend on the platform being used and are described here for Level 1 platforms. Users will need to adjust the instructions to match their machine configuration if their local machine is a Level 2-4 platform. 
 
-.. _SetUpPythonEnvC:
+.. _SetUpCont:
 
-Activate the Workflow
+Stage the Container 
 ------------------------
 
-Copy the container's modulefiles to the local working directory so that the files can be accessed outside of the container:
+To set up the container with your host system, run the ``stage-srw.sh`` script:
 
 .. code-block:: console
 
-   singularity exec -B /<local_base_dir>:/<container_dir> $img cp -r /opt/ufs-srweather-app/modulefiles .
-
-After this command runs, the local working directory should contain the ``modulefiles`` directory. 
-
-To activate the workflow, run the following commands: 
+   ./stage-srw.sh -c=<compiler> -m=<mpi_implementation> -p=<platform> -i=$img
 
-.. code-block:: console
+where:
 
-   module use /path/to/modulefiles
-   module load wflow_<platform>
+   * ``-c`` indicates the compiler on the user's local machine (e.g., ``intel/2022.1.2``)
+   * ``-m`` indicates the :term:`MPI` on the user's local machine (e.g., ``impi/2022.1.2``)
```

Comment:
@EdwardSnyder-NOAA In the Land DA PR, you listed a few more examples. Do those examples belong here as well?

Reply:
yeah, we could add those examples to here as well.

Reply:
added them